src/components/PageLayout/Comments.vue

```diff
@@ -44,7 +44,7 @@
   })
   export default class Comments extends Vue {
     @Prop({ default: [] })
-    private comments;
+    private comments!: Array<CommentItem>;
```

Comment:
`CommentItem[]` is what we've used in EternaJS, and I think it's much clearer in this case.

Reply:
changed